index.html

```diff
@@ -20,7 +20,7 @@ <h2 class="greetings">Glad to meet you!</h2>
                 <p>If you will like what you see and have a project you will need coded,don't hesitate to contact me</p> 
                 <p>look through some of my work and experince</p>
             </div>
-            <h3 class="heading" style="color:blue;">LET'S CONNECT</h3>
+            <h3 class="heading">LET'S CONNECT</h3>
             <div class="icon-bar">
```

Comment:
Kindly use a list for the icons. A div is a general element and not semantically correct at this point.